pkg/services/ngalert/notifier/channels/webhook.go

```diff
@@ -190,8 +190,13 @@ func (wn *WebhookNotifier) Notify(ctx context.Context, as ...*types.Alert) (bool
 		headers["Authorization"] = fmt.Sprintf("%s %s", wn.settings.AuthorizationScheme, wn.settings.AuthorizationCredentials)
 	}
 
+	parsedURL := tmpl(wn.settings.URL)
+	if tmplErr != nil {
```

Comment:
We should reset `tmplErr` to nil before this otherwise we could catch failures on the `DefaultMessageEmbed` or `DefaultMessageTitleEmbed` by mistake.

Reply:
Good catch!
I've already said this, but I'm not a fan of this type of error handling (-_-)